### PR TITLE
SEARCH-1342 | update schema versions

### PR DESCRIPTION
**Pull request by bradgarropy**
## Description

Update the `SearchInput` and `SearchResults` schemas to the latest version.

## Related Issue

[SEARCH-1342](https://jira.corp.magento.com/browse/SEARCH-1342)

## Motivation and Context

Necessary to support required fields in data modeling.

## How Has This Been Tested?

Tested with `jest` and the `example` directory while monitoring Snowplow events in Kibana.

## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

-   [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
-   [x] My code follows the code style of this project.
-   [ ] My change requires a change to the documentation.
-   [ ] I have updated the documentation accordingly.
-   [x] I have read the **CONTRIBUTING** document.
-   [x] I have added tests to cover my changes.
-   [x] All new and existing tests passed.

### CODE REVIEW

**Comment by bradgarropy** [comment]
The test failure is expected until the `magento-storefront-events-sdk` PRs get merged and released.

File: src/contexts/search/searchResults.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ const createContext = (searchResults?: SearchResults): SearchResultsContext => {
     const context = {
         schema: schemas.SEARCH_RESULTS_SCHEMA_URL,
         data: {
+            searchRequestId: "abc123",
```
Comment (evargast) [inline src/contexts/search/searchResults.ts:17]: 🤔  do we want this id hardcoded here? Shouldn't it also be  prop?

File: src/contexts/search/searchInput.ts [hunk 2]
```diff
@@ -7,19 +7,21 @@ import mse from "@adobe/magento-storefront-events-sdk";
...
 const createContext = (searchInput?: SearchInput): SearchInputContext => {
     const searchInputCtx = searchInput ?? mse.context.getSearchInput();
 
     const context = {
         schema: schemas.SEARCH_INPUT_SCHEMA_URL,
         data: {
-            source: searchInputCtx.source ?? undefined,
+            source: searchInputCtx.source ?? null,
+            queryType: "all",
+            searchRequestId: "abc123",
```
Comment (evargast) [inline src/contexts/search/searchInput.ts:20]: same comment

File: src/contexts/search/searchResults.ts (hunk 1 above)
Comment (evargast) [reply src/contexts/search/searchResults.ts:17]: nvm I just saw your other PR

**Pull request merged**